src/babel/babel-plugin.js

```diff
@@ -20,6 +27,9 @@ module.exports = function(babel, options = {}) {
   const {types: t} = babel
   const plugin = {
     name: '@effector/babel-plugin',
+    pre() {
+      this.importedNames = new Map()
```

Comment:
why `this`?

Reply:
https://github.com/jamiebuilds/babel-handbook/blob/master/translations/en/plugin-handbook.md#-pre-and-post-in-plugins